physics/module_mp_thompson.F90

```diff
@@ -5846,6 +5856,7 @@ subroutine calc_effectRad (t1d, p1d, qv1d, qc1d, nc1d, qi1d, ni1d, qs1d,   &
          endif
          lamc = (nc(k)*am_r*g_ratio(inu_c)/rc(k))**obmr
          re_qc1d(k) = SNGL(0.5D0 * DBLE(3.+inu_c)/lamc)
+         if (lsml .ne. 1) re_qc1d(k) = max(re_qc1d(k), 7.0E-6)
```

Comment:
Note. There's no need to be explicit about the precision here, since the precision is not parameterized in this routine.